client/src/featureform/resources.py

```diff
@@ -2250,11 +2311,51 @@ def config(self):
 def _get_and_set_equivalent_variant(
     resource_variant_proto, variant_field, stub
 ) -> Optional[str]:
+    rv_proto = None
     if feature_flag.is_enabled("FF_GET_EQUIVALENT_VARIANTS", True):
         # Get equivalent from stub
-        equivalent = stub.GetEquivalent(
-            pb.ResourceVariant(**{variant_field: resource_variant_proto})
-        )
+        if variant_field == "source_variant":
```

Comment:
why did we revert it to this?

Reply:
This is the thing to look ofr in this review, I think Riddhi may have made some minor changes to things that she didn't realize or catch go through this with a fine tooth comb

Reply:
Because we changed the argument from ResourceVariant to ResourceVariantRequest, we needed to explicitly write the variables, source_variant etc

Reply:
What am I supposed to do instead?

Reply:
the way it was before